appium/webdriver/webdriver.py

```diff
@@ -1412,7 +1412,20 @@ def finger_print(self, finger_id):
     # pylint: disable=protected-access
 
     def _addCommands(self):
-        super(WebDriver, self)._addCommands()
+        self.command_executor._commands[Command.CONTEXTS] = \
```

Comment:
What is the reason to remove these from extensions?

Reply:
As below, the missing value happened. https://travis-ci.org/appium/python-client/jobs/469424297

```
self = <selenium.webdriver.remote.remote_connection.RemoteConnection object at 0x7f6dd07009b0>
command = ('getCurrentContext',), params = {'sessionId': '1234567890'}
    def execute(self, command, params):
        """
        Send a command to the remote server.
    
        Any path subtitutions required for the URL mapped to the command should be
        included in the command parameters.
    
        :Args:
         - command - A string specifying the command to execute.
         - params - A dictionary of named parameters to send with the command as
           its JSON payload.
        """
>       command_info = self._commands[command]
E       KeyError: ('getCurrentContext',)
.tox/py37/lib/python3.7/site-packages/selenium/webdriver/remote/remote_connection.py:367: KeyError
```

`('GET', '/session/$sessionId/contexts')` did not stored in `_commands` correctly.
(Thus, I reverted it as a hot fix)

Reply:
sorry for that. I have created a better fix: https://github.com/appium/python-client/pull/297/files

Reply:
this multiple inheritance stuff is not so simple though %)